Hyperledger_Fabric_Technical_Charter.md

```diff
@@ -2,158 +2,133 @@
 
-    2. **Scope:** This process applies to significant ecosystem-wide matters, such as: (1) accepting or archiving Sub-Projects, (2) major changes to core principles or guidelines, (3) adopting or significantly altering other ecosystem-wide policies.
+        4. establishing community norms, workflows, issuing releases, and security issue reporting policies; and
 
-    3. **Ecosystem-Wide Voting Process**
+        5. approving and implementing policies and processes for contributing (to be published in the CONTRIBUTING file) and coordinating with the series manager of the Project (as provided for in the Series Agreement, the “Series Manager”) to resolve matters or concerns that may arise as set forth in Section 7 of this Charter.
 
-        1.  **Initiation**: Proposals for ecosystem-wide decisions are reviewed by the TSC. If a formal vote is deemed necessary, the TSC initiates it, putting the matter to the TSC Members affiliated with all recognized and active Sub-Projects for their respective Sub-Project's collective vote.
+4. **EC/TSC Voting**
 
-        2.  **Sub-Project Collective Vote Determination:**
-            
-            1. Each active Sub-Project contributes one collective vote (approve/disapprove) to an ecosystem-wide decision. This vote is determined by the TSC Members affiliated with that Sub-Project.
-            
-            2. **Internal Voting Eligibility:** Only the TSC Members affiliated with the Sub-Project are eligible to vote in the internal process that determines the Sub-Project's collective stance.
+    1. While the Project aims to operate as a consensus-based community, if any decision of any TSC or EC requires a vote to move the matter  forward, the voting members of the EC or a TSC, as applicable, will vote on a one vote per voting member basis.
 
-            3. **Internal Decision Mechanism:** Typically, a **Sub-Project's** collective vote is 'approve' if a majority of its participating affiliated TSC members vote to approve the proposal; otherwise, its collective vote is 'disapprove'. Alternatively, the **Sub-Project** may use its own documented procedures for its affiliated TSC members to determine its collective vote (approve/disapprove).
+    2. Quorum for EC and TSC meetings requires at least fifty percent of all voting members of the EC or a TSC, as applicable, to be present. The EC or a TSC, as applicable, may continue to meet if quorum is not met but will be prevented from making any decisions at the meeting.
 
-        3.  **Submission & Tally:** Each group of TSC Members affiliated with a Sub-Project submits their Sub-Project's collective vote to the overall TSC.
+    3. Except as provided in Section 8.c. and 9.a, decisions by vote at a meeting require a majority vote of those in attendance, provided quorum is met. Decisions made by electronic vote without a meeting require a majority vote of all voting members of the TSC.
```

Comment:
When rendered in GitHub, section 8.c appears as 8.iii (a.k.a. 8.3), and 9.a appears as 9.i (a.k.a. 9.1).

Reply:
true, it should be changed to 8.iii and 9.i